Description: Arch: Ignore grammer pickle mis-matches in lib2to3.
 Pickle files encode the endian of the arch that built them.  They are
 architecture-independent, but there isn't a canonical endianness, both are
 handled on load.

Forwarded: no

--- a/Lib/lib2to3/pgen2/driver.py
+++ b/Lib/lib2to3/pgen2/driver.py
@@ -119,7 +119,10 @@ def load_grammar(gt="Grammar.txt", gp=No
     if force or not _newer(gp, gt):
         logger.info("Generating grammar tables from %s", gt)
         g = pgen.generate_grammar(gt)
-        if save:
+        # the pickle files mismatch, when built on different architectures.
+        # don't save these for now. An alternative solution might be to
+        # include the multiarch triplet into the file name
+        if False:
             logger.info("Writing grammar tables to %s", gp)
             try:
                 g.dump(gp)
